### PR TITLE
Several template fixes

### PR DESCRIPTION
**Pull request by emanuelschuetze**

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@normanjaeckel ping

File: openslides/agenda/signals.py [hunk 1]
```diff
@@ -24,23 +24,60 @@ def setup_agenda_config(sender, **kwargs):
     grouped. This function connected to the signal
```
Comment (normanjaeckel) [inline openslides/agenda/signals.py]: Update docstring. Now we have subgroups so the docstring is outdated.

Comment (emanuelschuetze) [inline openslides/agenda/signals.py]: Done.

**Comment by emanuelschuetze** [comment]
2 comments fixed and rebased.

**Pull request merged**